pkg/flowaggregator/flowaggregator.go

```diff
@@ -360,20 +328,28 @@ func (fa *flowAggregator) initExportingProcess() error {
 		return fmt.Errorf("got error when initializing IPFIX exporting process: %v", err)
 	}
 	fa.exportingProcess = ep
-	// Currently, we send 4 templates for covering all the cases in dual-stack clusters, where Pod IPs
-	// and original exporter IP could belong to different IP families.
-	if fa.templateIDv4Expv4, err = fa.createAndSendTemplate(false, false); err != nil {
-		return err
-	}
-	if fa.templateIDv4Expv6, err = fa.createAndSendTemplate(false, true); err != nil {
-		return err
-	}
-	if fa.templateIDv6Expv4, err = fa.createAndSendTemplate(true, false); err != nil {
```

Comment:
I feel like even with 2 templates only, the helper function may still be a good idea?

Reply:
Done. Thanks!